src/CoreWCF.Templates/src/templates/CoreWCFService/.template.config/template.json

```diff
@@ -0,0 +1,189 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "CoreWCF",
```

Comment:
@mconnew author is displayed in VS new project wizard.

Reply:
I would suggest either:
* CoreWCF Project 
* or if it doesn't balk at having a URL as the author, then https://github.com/CoreWCF/CoreWCF

Reply:
I used https://github.com/CoreWCF as author.